README.md

```diff
@@ -1,23 +1,26 @@
 # UI Kit
 
-> UI Kit for react-native and react-native-web apps with modal and drawer
-
-[UIKit Storybook Link](https://react-native-web-uikit.storybook.trucknet.io)
-
-![Modal/Drawer](docs/uikit_storybook.gif)
+> UI Kit for react-native and react-native-web
 
 ## StoryBook
 
-    - git clone git@github.com:trucknet-io/react-native-web-uikit.git
-    - npm install
+- `git clone git@github.com:trucknet-io/react-native-web-uikit.git`
+- `npm install`
+- for adding story name file `ComponentName.story.tsx` in component folder
```

Comment:
to add a Storybook story, create a `ComponentName.story.tsx` file in component's folder